pkg/build_zip.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This tool builds zip files from a list of inputs."""
+
+import sys
+
+from absl import flags
+from helpers import UnquoteAndSplit
+from zipfile import ZipFile, ZipInfo, ZIP_DEFLATED
+
+flags.DEFINE_string('output', None, 'The output file, mandatory')
+flags.mark_flag_as_required('output')
+
+flags.DEFINE_multi_string('file', [], 'A file to add to the zip')
+
+FLAGS = flags.FLAGS
+
+def main(unused_argv):
```

Comment:
I didn't go all out making this a class like `pkg_tar` (I assume it's like that for testing/reusing the impl for other `pkg_*` formats?) let me know if you'd like to see anything like this.

Reply:
We will probably be there at some point. But you can figure out the API when needed rather than desigining in a vacuum.